echo.js

```diff
@@ -1,7 +1,25 @@
-function echo(){
-  // ...
-}
+onload = function() {
+  // echo function is to print out each argument on a new line
+  function echo() {
 
-echo();
-echo('bla');
-echo('foo', 'bar', 'baz');
+    var createElement = document.createElement('p');
+    var innerContent = [];
```

Comment:
Why make this an array? Do you ever need to reference more than the current element's `innerContent`?